agent-discoveries-frontend/src/App.tsx

```diff
@@ -2,20 +2,16 @@ import reactLogo from './assets/react.svg'
 import './App.css'
 import Login from './Views/Login/Login'
 import HomePage from './Views/Home/HomePage'
+import ProfilePage from './Views/ProfilePage/ProfilePage'
+
 import { useState } from 'react'
 
 function App() {
-  const [isLoggedIn, setIsLoggedIn] = useState(false)
+  const [isLoggedIn, setIsLoggedIn] = useState(true)
```

Comment:
Why has this changed?

Reply:
We will change it back; we changed it to true to make the user logged in.